### PR TITLE
Extend pool and loan pallet events

### PR DESCRIPTION
**Pull request by offerijns**
- Extend order events to include old & new order amounts
- Add event on tranche rebalancing
- Extend loan pricing event with pricing info

### CODE REVIEW

File: pallets/pools/src/lib.rs [hunk 1]
```diff
@@ -701,12 +707,13 @@ pub mod pallet {
...
-			amount: T::Balance,
+			new_order: T::Balance,
 		) -> DispatchResult {
 			let who = ensure_signed(origin)?;
 
-			let tranche_id =
-				Pool::<T>::try_mutate(pool_id, |pool| -> Result<T::TrancheId, DispatchError> {
+			let (tranche_id, old_order) = Pool::<T>::try_mutate(
+				pool_id,
+				|pool| -> Result<(T::TrancheId, T::Balance), DispatchError> {
```
Comment (NunoAlexandre) [inline pallets/pools/src/lib.rs:716]: Do you think we could move these mutations to a separate function? That would keep the high-level extrinsic more declarative, easier to follow and less noisy.

Comment (offerijns) [reply pallets/pools/src/lib.rs:716]: I think in this case, we already have sufficient separation of code. This method now calls into `do_update_invest_order`, which on its own calls into `update_order_amount`. I think adding another level will just make it harder to understand.

Comment (NunoAlexandre) [reply pallets/pools/src/lib.rs:716]: Fair 💪🏻 I was also reviewing this on the train with a smaller screen so everything looked more packed and harder to follow.

File: pallets/loans/src/lib.rs [hunk 2]
```diff
@@ -200,8 +200,13 @@ pub mod pallet {
...
 		Closed(PoolIdOf<T>, T::LoanId, AssetOf<T>),
-		/// A loan was priced. [pool, loan]
-		Priced(PoolIdOf<T>, T::LoanId),
+		/// A loan was priced. [pool, loan, interest_rate_per_sec, loan_type]
+		Priced(
+			PoolIdOf<T>,
+			T::LoanId,
+			T::Rate,
+			LoanType<T::Rate, T::Amount>,
+		),
```
Comment (mustermeiszer) [inline pallets/loans/src/lib.rs:209]: Nice!

**Review by mustermeiszer — APPROVED** [review]
LGTM!

**Pull request merged**